browser/src/map/handler/Map.Keyboard.js

```diff
@@ -497,6 +497,21 @@ L.Map.Keyboard = L.Handler.extend({
 			return;
 		}
 
+		if (!this.modifier && this._map.getDocType() === 'spreadsheet' && (keyCode === this.keyCodes.pageUp || keyCode === this.keyCodes.pageDown) && ev.type === 'keydown') {
+			ev.preventDefault();
+			var currentSelectedPart = this._map._docLayer._selectedPart;
+			var parts = this._map._docLayer._parts;
+			var partToSelect = 0;
+			this._map._docLayer._clearReferences();
+
+			if (keyCode === this.keyCodes.pageUp) {
+				partToSelect = currentSelectedPart != parts - 1 ? currentSelectedPart + 1 : 0;
+			}
+			else {
+				partToSelect = currentSelectedPart != 0 ? currentSelectedPart - 1 : parts - 1;
+			}
+			this._map.setPart(parseInt(partToSelect), /*external:*/ false, /*calledFromSetPartHandler:*/ true);
+		}
```

Comment:
you should do early return here so you will not enter into code block which scrolls the page (it's somewhere below)

Reply:
i will add that, Thanks :)